ui/App.js

```diff
@@ -11,7 +11,7 @@ export default function App() {
         { value: 'goerli', label: 'Görli' }
     ]
 
-    if (process.env.NODE_ENV === 'testing'){
+    if (process.env.TESTING === 'true'){
```

Comment:
WDYT about using `||` here instead of getting rid of NODE_ENV altogether? When I'm testing locally the current setup works and I think the `TESTING` is not visible or will need additional configuration...